NOTICE

```diff
@@ -7,17 +7,11 @@ The Apache Software Foundation (http://www.apache.org/).
 This project bundles the following dependencies under the MIT license.
 See bundled license files for details.
 
-- jQuery v3.5.1 | (c) JS Foundation and other contributors
-	-> in "docs/page/js/jquery.min.js"
 - AnchorJS v3.1.0 (https://github.com/bryanbraun/anchorjs) Copyright (c) 2016 Bryan Braun
-    -> in "docs/page/js/anchor.min.js"
-- Bootstrap v3.3.4 (http://getbootstrap.com) Copyright 2011-2015 Twitter, Inc.
-    -> css in "docs/page/css/bootstrap.min.css"
-    -> js in "docs/page/js/bootstrap.min.js"
-    -> fonts in "docs/page/fonts"
+    -> in "docs/static/js/anchor.min.js"
 
-Note: The "docs/page/fonts/glyphicons-halflings*" fonts are included in Bootstrap, and licensed under the same license as Boostrap.
-See https://glyphicons.com/old/license.html#old-halflings-bootstrap for more information.
+- hugo-book (https://github.com/alex-shpak/hugo-book) Copyright (c) 2018 Alex Shpak
+    -> in "docs/themes/book"
 
```

Comment:
```suggestion
```
This is not included in the source release.

Reply:
Got it. I can delete these two lines. 
> I saw jquery and bootstrap were also only used and referenced in Flink doc so I intended to put hugo-book here (I will correct this).